site/src/tokens/layout.md

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
Why this one was removed?

Reply:
I had originally removed it because it didn't seem to contain much useful information. That said, I've restored the file because I don't think I really _needed_ to remove it.